packages/backup-restore-operator/charts/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: backup-restore-operator-installer
+subjects:
+- kind: ServiceAccount
+  name: backup-restore-operator-serviceaccount
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
We should create a ClusterRole CR to this and add a label to aggregate to `cluster-admin` instead of binding directly to the Kubernetes `cluster-admin`.

Here's an example from `rancher-monitoring`:

https://github.com/rancher/charts/blob/98d70cee35a19afd217633b5283e21a0dd0b8296/packages/rancher-monitoring/rancher-monitoring.patch#L1170-L1207

Reply:
This operator should be able to modify/create/delete any resources, and the ClusterRole from monitoring example specifies rules on particular resources. Since we want it to be able to target resources from all apigroups, I don't see how we'll know what resources to add to this clusterRole